index.rst

```diff
@@ -367,6 +449,9 @@ While having native file system support on the user's system would be extremely
 
 None of the other options seem sufficiently compelling over the availability and well-understood features of NFSv3.
 
+Remote access to storage
+^^^^^^^^^^^^^^^^^^^^^^^^
+
 This leaves the question of how to provide file system access from a user's local device.
```

Comment:
Do you want to mention synchronization in addition to access?  The "I want to edit remotely" use case does basically presume an implementation that provides synchronization.  But maybe we don't have to / want to raise this detail at this point.